projects/BFDR/IJEFetalDeath.cs

```diff
@@ -3846,12 +3846,11 @@ public string MOM_OC_T
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: 
-                return "";
+                return record.MotherOccupation;
```

Comment:
Is there a reason you're not using `LeftJustified_Set`/`LeftJustified_Get` for these instead of setting them directly?

Reply:
I was following precedent in IJEBirth. It probably makes sense to change there as well.

Reply:
Yes, they would need to be truncated to fit into IJE otherwise the formatting in IJE would be incorrect since its all based on character position. If the intention was for the profile to specify the same length requirement as IJE, then the string length should be changed to 40 in the IG. If the string length 40 was chosen for another reason, it can stay as is in the IG.

Reply:
LeftJustified_Get will throw validation errors when it reads in a FHIR record. I'm assuming if you have have a FHIR record with a string longer than expected, conversion to IJE should throw these errors rather than the fields being truncated. That said, I will add truncation to the setters for the IJE fields themselves.